### PR TITLE
extract populate weight collection.

### PR DESCRIPTION
**Pull request by alexpantyukhin**

### CODE REVIEW

File: tensorflow/python/feature_column/feature_column.py [hunk 1]
```diff
@@ -174,11 +174,7 @@ def _internal_input_layer(features,
           'Items of feature_columns must be a _DenseColumn. '
           'You can wrap a categorical column with an '
           'embedding_column or indicator_column. Given: {}'.format(column))
-  weight_collections = list(weight_collections or [])
-  if ops.GraphKeys.GLOBAL_VARIABLES not in weight_collections:
-    weight_collections.append(ops.GraphKeys.GLOBAL_VARIABLES)
-  if ops.GraphKeys.MODEL_VARIABLES not in weight_collections:
-    weight_collections.append(ops.GraphKeys.MODEL_VARIABLES)
+  weight_collections = _populate_weight_collections(weight_collections)
```
Comment (ispirmustafa) [inline tensorflow/python/feature_column/feature_column.py]: this name of the function confuses the reader. please either dissolve this function or find a better name. may be: '_append_global_and_model_variables_collections(weight_collections)'

Comment (alexpantyukhin) [reply tensorflow/python/feature_column/feature_column.py]: @ispirmustafa Thanks for review!
I found that duplication and thought that these actions could be named and moved into a function.
Yes, you are right about the name. The `populate_weight_collection` is confusable. I like your variant with `append_global_and_models_variables_collections` but it looks like it describes all actions inside the function. What is to have something like `_append_variables_to_weight_collections`? It just says that some variables will be added, and what are variables are listed inside the body. What do you think?

Comment (ispirmustafa) [reply tensorflow/python/feature_column/feature_column.py]: Based on this discussion, IMHO we should not wrap them into a function. I mean let's cancel this PR.

Comment (alexpantyukhin) [reply tensorflow/python/feature_column/feature_column.py]: @ispirmustafa okay

**Pull request closed without merging**